draft-ietf-rats-architecture.md

```diff
@@ -376,6 +376,73 @@ create Evidence. Examples of environments that can be used as
 Attesting Environments include Trusted Execution Environments (TEE),
 embedded Secure Elements (eSE), or Hardware Security Modules (HSM).
 
+## Layered Attestation Procedures
+
+By definition, the Attester role takes on the duty to create Evidence.
+The fact that an Attester role is composed of several types of environments that
+can be nested or staged adds complexity to the architectural layout of how an
+Attester -- in itself -- is composed and therefore has to conduct the Claims collection
+in order to create believable Attestation Evidence. This complexity is
+independent of the concept defined by Composite Devices.
+Composite Attesters are inherently composed of multiple nested environments.
+The following example is intended to illustrate this composition:
+
+A very common example is elaborated on to illustrate Layered Attestation.
+
+The very first Attesting Environment has to ensure the integrity of
+the (U)EFI / BIOS / Firmware that initially boots up a composite device (e.g.,
+a cell phone). A hardware component, which effectively can encompasses
+immutable execution environments (static code Roots of Trust) and mutable execution environments (dynamic code Roots of Trusts),
+executes the very first Claim Collection. These Claims have to be stored
+securely. At this stage of the boot-cycle of a
+composite device, the Claims collected typically cannot be composed into Evidence.
+
+The very first Attesting Environment in this example can be a hardware component that is a Static Code Root of Trust.
+As in any other scenario, this hardware component is the first Attesting Environment.
+It collects a rather concise number of Claims about the Target Environment.
+The Target Environment in this example is the (U)EFI / BIOS / Firmware.
+After the boot sequence started, the Target Environment conducts the
+most important and defining feature of layered attestation:
+The successfully measured [comment: this is why Henk prefers 'measured' above 'collected'
+because it can be used in English text better] environment that is the
+(U)EFI / BIOS / Firmware now becomes the Attesting Environment. Analogously, the Attesting Environment hands off its duty to one of its Target Environments. This procedure in Layered Attestation is called Staging.
```

Comment:
I haven't heard the term Staging before, not sure why we need to mention it here